packages/imu_driver/src/imu_node.py

```diff
@@ -98,6 +129,29 @@ def publish_data(self, event):
             msg.linear_acceleration.x, msg.linear_acceleration.y, msg.linear_acceleration.z = tuple(
                 (acc_data[i] - self._accel_offset[i]) for i in range(len(acc_data)))
             msg.linear_acceleration_covariance = [0.0 for _ in range(len(msg.linear_acceleration_covariance))]
+
+            # perform static transfrom and change frame if applicable
+            msg.header.frame_id = FALLBACK_IMU_PUB_FRAME.format(robot_hostname=self._veh)
+            if self._static_transform is not None:
+                v3s = Vector3Stamped()
+                v3s.header = msg.header
+                v3s.vector = msg.linear_acceleration
+                msg.linear_acceleration = self._tf_listener.transformVector3(self._imu_pub_frame, v3s).vector
+
+                v3s = Vector3Stamped()
+                v3s.header = msg.header
+                v3s.vector = msg.angular_velocity
+                msg.angular_velocity = self._tf_listener.transformVector3(self._imu_pub_frame, v3s).vector
```

Comment:
While linear accelerations are not affected by a pure translation (because axis never change orientation), angular velocities do change with a pure translation. Given how the IMU is mounted, we expect the roll speed to be almost identical (maybe just a sign swap), while pitch and yaw speeds would change quite a bit given the distance between the origin of the `bottom_plate` reference frame and the mounting point of the IMU.

Reply:
The above code is not correct. But as discussed in the meeting, only rotation between frames needs to be applied, for both linear accel and angular vels.